src/ui/ViewModels/FromUcasViewModel.cs

```diff
@@ -17,6 +17,7 @@ public class VariantViewModel
         public CourseEnrichmentViewModel CourseEnrichment { get; set; }
         public CourseRouteDataViewModel RouteData { get; set; }
         public Uri LiveSearchUrl { get; set; }
+        public bool IsSalary { get; set; }
```

Comment:
`IsSalaried` would be better english I think

minor

Reply:
That bool will change all together